doc/source/parameters/dem/floating_mesh.rst

```diff
@@ -0,0 +1,51 @@
+Floating Mesh (Solid Objects)
+--------------------------------------
+Floating meshes (solid objects) are finite (limited) auxiliary objects that can be stationary or moving. Rotating impellers, sliding surfaces, and finite stoppers are examples of floating meshes. Note that the main differences between floating meshes and floating walls are: 1. floating wall is a surface, while a floating mesh can be a volume or a plane; 2. floating wall is infinite, while floating mesh is finite; 3. floating wall is stationary while floating mesh may be stationary or moving.
+
+.. note:: 
+    Solid objects (floating meshes) in Lethe have to be defined using triangular (simplex) meshes.
+
+This subsection explains the solid objects (floating meshes) information. First of all, the ``number of solids`` is specified. Then for each solid object, we need a ``mesh`` subsection. In these subsections, the ``type``, ``initial refinement``, and other properties of the objects are defined. Note that ``simplex`` must be ``true`` for all the objects, since the solid objects are defined using simplex meshes.
```

Comment:
```suggestion
This subsection explains the solid objects (floating meshes) information. First of all, the ``number of solids`` is specified. Then, for each solid object,  a ``mesh`` subsection must be included. In these subsections, the ``type``, ``initial refinement``, and other properties of the objects are defined. Note that ``simplex`` must be ``true`` for all the objects, since the solid objects are defined using simplex meshes.
```